index.css

```diff
@@ -76,3 +74,162 @@ body {
 .social-icons li {
   list-style-type: none;
 }
+
+/* ----------Works page styling begins here----- */
+
+.works-card {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: center;
+  margin: 8em 7% 0 7%;
+  padding: 1em 1em 2em 1em;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  color: #172b4d;
+  background-color: fff;
+}
+
+.works-title {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+}
+
+.first_buttons {
+  display: flex;
+  flex-direction: row;
+  gap: 10px;
+  padding-top: 10px;
+  justify-content: flex-start;
+}
+
+.first_buttons a {
+  color: #6070ff;
+  background-color: #ebebff;
+  border-radius: 8px;
+  text-decoration: none;
+  padding: 4px 12px;
+}
+
+.card_details p {
+  width: 95%;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.card_details ul {
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+  font-weight: 600;
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  gap: 20px;
+}
+
+.canopy {
+  color: #344563;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  list-style: none;
+}
+
+.works-text {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+}
+
+.last-button {
+  display: flex;
+  flex-direction: row;
+  padding-top: 24px;
+  padding-bottom: 16px;
+}
+
+.first_buttons,
+.coding-languages ul {
+  list-style: none;
+}
+
+/* -------------- About page styling begins here -------- */
+
+.about_page {
+  height: 100vh;
+}
+
+.works-card-2 {
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  margin: 8em 7% 0 7%;
+  padding: 1em 1em 2em 1em;
+  border: 1px solid #dfe1e6;
+  border-top-right-radius: 100px;
+}
+
+button {
+  color: #396df2;
+  line-height: 24px;
+  font-size: 17px;
+  font-weight: 500;
+  border-radius: 8px;
+  text-decoration: none;
+  padding: 12px 12px;
+  background-color: #fff;
+  border: 1px solid #396df2;
+}
+
+.last-button button {
+  color: #396df2;
+  line-height: 24px;
+  font-size: 17px;
+  font-weight: 500;
+  border-radius: 8px;
+  text-decoration: none;
+  padding: 12px;
+  background-color: #fff;
+  border: 1px solid #396df2;
+}
+
+.icon_one {
+  width: 90%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  gap: 11pc;
+  margin: 30px 10px;
+}
+
+.icon_one h4 {
+  font-size: 20px;
+  line-height: 24px;
+  font-weight: 500;
+}
+
+.languages_details {
+  display: flex;
+  align-items: center;
+  border-radius: 8px;
+  padding: 10px;
+  background-color: #f7f7f9;
+  gap: 12px;
+  margin: 12px;
+}
+
+.languages_wrapper {
+  padding: 12px 24px;
+}
```

Comment:
- [ ] This section of languages and Skills is not consistent with Figma design and overlapping. kindly consider changing it to make it similar to the Figma design as it is a project requirement.
your design
![zx](https://user-images.githubusercontent.com/96683843/212406272-b15c38d9-67bc-45a2-99b4-0b8fb242e88b.jpg)


figma design
![zxcv](https://user-images.githubusercontent.com/96683843/212406292-6535fb8c-c098-45dc-91c2-561beedb53dc.jpg)